ipatests/test_integration/test_sso.py

```diff
@@ -98,3 +133,17 @@ def test_sso_login_with_ipa_user(self):
         username_fl = 'test user'
         password = self.keycloak.config.admin_password
         keycloak_login(self.keycloak, username, password, username_fl)
+
+    def test_ipa_login_with_sso_user(self):
+        """
+        Test case to check authenticating to IPA as Keycloak user
```

Comment:
Could you please add related pagure link.
And add more detailed description about test.

Reply:
Updated the docstring test description to be more detailed.

The pagure ticket link is in the commit message.  Should I include it in the test's docstring too?

Reply:
Added related link in docstring in last update